src/Components/TripETA/_trip-eta.scss

```diff
@@ -46,4 +46,36 @@
     color: rgb(255, 255, 255);
     background-color: rgb(116, 187, 235);
   }
-}
\ No newline at end of file
+}
+
+@media screen and (max-width: 360px) {
+
+  .eta-modal a {
+    height: 17%;
+    width: 35%;
+  }
+
+  .eta-message {
+    font-size: 1.48rem;
+  }
+
+  .eta-message span:nth-of-type(1n) {
+      font-size: 1em;
+      margin:0.4em
+    }
+
+  .eta-message span:nth-of-type(2n) {
+    font-size: 1.15em;
+    font-weight: 500;
+    margin: 0.3em;
+  }
+
+  .begin-trip-btn {
```

Comment:
I'm happy to take a look at button size responsiveness in the morning - just remind me which specifically we need to adjust.